### PR TITLE
added index priority support for ISM

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/ism/policies.md [hunk 1]
```diff
@@ -321,6 +322,26 @@ Parameter | Description | Type | Required | Default
...
 ```
 
+### index_priority
+
+Set the priority for the index in a specific state. Unallocated shards of indices are recovered in the order of their priority, whenever possible.
+
+The `index_priority` operation has the following parameter:
+
+Parameter | Description | Type | Required | Default
+:--- | :--- |:--- |:--- |
```
Comment (bowenlan-amzn) [inline docs/ism/policies.md:332]: it seems we miss another `:---` here, the table is not rendered out correctly, and please check there are other not rendered table in this `policies.md` page

File: docs/ism/policies.md [hunk 2]
```diff
@@ -321,6 +322,26 @@ Parameter | Description | Type | Required | Default
 }
 ```
 
+### index_priority
+
+Set the priority for the index in a specific state. Unallocated shards of indices are recovered in the order of their priority, whenever possible.
```
Comment (aetter) [inline docs/ism/policies.md:327]: By default, what priority (if any) do indices have? For priority, would 1 be extremely high or extremely low?

File: docs/ism/policies.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/ism/policies.md]: changed.

File: docs/ism/policies.md (hunk 2 above)
Comment (ashwinkumar12345) [reply docs/ism/policies.md]: Added the default priority and some context. Please review.

**Pull request merged**